inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/MonitoringPage.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
```

Comment:
Let's rename it from "MonitoringPage" to "DynamicWorkloadManagementPage" or something like that - we already have the "MonitoringPage" - it is a bit confusing.